### PR TITLE
Create custom authoring pages

### PR DESCRIPTION
**Pull request by sfentress**
This creates the authoring pages at

* https://connected-bio-spaces.concord.org/branch/create-custom-authoring-pages/mice-authoring.html and
* https://connected-bio-spaces.concord.org/branch/create-custom-authoring-pages/peas-authoring.html

### CODE REVIEW

**Review by mklewandowski — APPROVED** [review]
👍 This looks great!  I went through a bunch of combinations of settings on both mice and peas and everything seems to be working!

**Comment by sfentress** [comment]
There are some lingering additional stories that have been added for the authoring, and at least one small issue that has come up in QA, but I'm going to merge this in to make the next merge smaller.

File: package.json [hunk 1]
```diff
@@ -71,7 +71,7 @@
     "@types/lodash": "^4.14.116",
     "@types/query-string": "^6.1.0",
     "@types/rc-slider": "^8.6.5",
-    "@types/react": "^16.4.14",
+    "@types/react": "^16.14.11",
     "@types/react-dom": "^16.0.8",
```
Comment (kswenson) [inline package.json:75]: Should bump version of `@types/react-dom` as well.

File: src/components/authoring-common.tsx [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+    errors.numHawks = "Number of hawks must be between 0 and 10";
+  }
+
+  if (values.populations!.inheritance!.chanceOfMutations! < 0 ||
+      values.populations!.inheritance!.chanceOfMutations! > 100) {
+    errors.popChanceOfMutations = "Chance of mutations must be between 0 and 100";
+  }
+
+  if (values.populations!.initialPopulation!.white! + values.populations!.initialPopulation!.tan! > 100) {
+    errors.initialPopulation = "First two percentaged can't sum to more than 100%";
```
Comment (kswenson) [inline src/components/authoring-common.tsx:54]: "percentaged" => "percentages"

File: src/components/authoring-common.tsx [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+  }
+
+  if (values.breeding!.chanceOfMutations! < 0 ||
+      values.breeding!.chanceOfMutations! > 100) {
+    errors.breedingChanceOfMutations = "Chance of mutations must be between 0 and 100";
+  }
+
+  if (!values.breeding!.enableColorChart! &&
+    !values.breeding!.enableGenotypeChart! && !values.breeding!.enableSexChart!) {
+    errors.oneBreedingChart = "At least on graph must be selected";
```
Comment (kswenson) [inline src/components/authoring-common.tsx:69]: "on" => "one"

**Pull request merged**